tests/Database/DatabaseEloquentIntegrationTest.php

```diff
@@ -768,7 +769,7 @@ protected function connection($connection = 'default')
     /**
      * Get a schema builder instance.
      *
-     * @return Schema\Builder
+     * @return Illuminate\Database\Schema\Builder
```

Comment:
I added it, but do you want leading slashes when the class is not in a namespace?